pinecone/models.go

```diff
@@ -112,4 +112,4 @@ type Usage struct {
 }
 
 type Filter = structpb.Struct
-type Metadata = structpb.Struct
+type Metadata = structpb.Struct
```

Comment:
accidental change?

Reply:
I think it was just auto-formatting removing the line at the end of the file. Is that an issue here?